README.md

```diff
@@ -6,7 +6,6 @@
 [![Server Test](https://github.com/saiya/dsps/workflows/Server%20Test/badge.svg?1)](https://github.com/saiya/dsps/actions?query=workflow%3A%22Server+Test%22)
```

Comment:
I found some other badges needed to update.

for example
```suggestion
[![Server Test](https://github.com/m3dev/dsps/workflows/Server%20Test/badge.svg?1)](https://github.com/m3dev/dsps/actions?query=workflow%3A%22Server+Test%22)
```

Reply:
@hirosassa Sorry, I also fixed them.

`Go Report Card` results in error now, but I assume this will be fixed after we make first release in this repo(please tell me if i'm wrong)

Reply:
this needs manually input the form below and generated the URL

https://goreportcard.com/

Reply:
Thank you. I already tried it, but GoReportCard showed `There was an error processing your request: Could not analyze the repository: no .go files found` error...

![image](https://user-images.githubusercontent.com/26357448/176328976-9bf3b59a-44b7-485e-9f7f-a90d4d775a47.png)

Reply:
@to-lz1 You can use this! (you should need to add `/server`)
```
[![Go Report Card](https://goreportcard.com/badge/github.com/saiya/dsps/server)](https://goreportcard.com/report/github.com/saiya/dsps/server)
```